### DISCUSSION TITLE
Não carrega save

### DISCUSSION THREAD

**Discussion by Lencaster** [mupen64plus-ae/mupen64plus-ae#1092] [General]
Durante o jogo paper mario versão europeia, o jogo trava sempre em um lugar diferente e não tem como carregar os saves em nenhum modo.

**Comment by fzurita** [comment]
Try this: Select the game, go to settings, and change the emulation profile to GLideN64-Medium.

Tente o seguinte: selecione o jogo, vá para as configurações e altere o perfil de emulação para GLideN64-Medium.
